app/src/components/market/common/history_chart/chart.tsx

```diff
@@ -110,7 +110,7 @@ const Legend = styled.li`
   }
 `
 
-const AnEvenSmallerLittleBall = styled(OutcomeItemLittleBallOfJoyAndDifferentColors)`
+const AnEvenSmallerLittleBall = styled(OutcomeItemLittleBallOfJoyAndDifferentColors as any)`
```

Comment:
Errors about this being a reference loop in the compilation occur seemingly randomly with the new TS (passing in `styled(styled(stuff))` causes some compilation to fail sometimes...)

Reply:
Okay probably should `s/styled\((\w+) as any\)/styled($1)` after upgrading styled-components is the proper fix here... https://github.com/microsoft/TypeScript/issues/37597

Reply:
Should we add the `styled-components.d.ts` file to the project directly, as per: https://github.com/microsoft/TypeScript/issues/37597#issuecomment-628149946?

Reply:
I would check if the new type definitions have been merged in before before correcting the issue.